netbout-web/src/test/java/com/netbout/rest/TkInboxTest.java

```diff
@@ -94,14 +94,14 @@ public void searchesBouts() throws Exception {
         final Aliases aliases = base.user(new URN(urn)).aliases();
         aliases.add("test2");
         final Inbox inbox = aliases.iterate().iterator().next().inbox();
-        final Bout firstBout = inbox.bout(inbox.start());
-        final String firstTitle = "bout1 title";
-        firstBout.rename(firstTitle);
-        final Bout secondBout = inbox.bout(inbox.start());
-        final String secondTitle = "bout2 title";
-        secondBout.rename(secondTitle);
-        firstBout.messages().post("hello");
-        secondBout.messages().post("world");
+        final Bout firstbout = inbox.bout(inbox.start());
```

Comment:
actually there are 4 variables, firstBout and firstTitle, secondBout and secondTitle.